test/extensions/filters/listener/original_dst/original_dst_fuzz_test.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.listener.original_dst.v3;
+
+import "envoy/extensions/filters/listener/original_dst/v3/original_dst.proto";
+import "google/protobuf/empty.proto";
+import "validate/validate.proto";
+
+message OriginalDstTestCase {
+  envoy.extensions.filters.listener.original_dst.v3.OriginalDst config = 1;
+  string address = 2;
```

Comment:
The address proto seems more restrictive than necessary (the core address string field is the same, but an Address also requires you to specify a protocol and port, along with several other new fields). I don't think I would actually be able to use any of the additional fields, since those properties are automatically resolved by network utility functions from the address. It also might make more sense to allow the fuzzer to ingest a wider set of (possibly empty or invalid) addresses?